bindings/rust/wasmedge-sys/build.rs

```diff
@@ -174,6 +174,16 @@ fn find_wasmedge() -> Option<Paths> {
             lib_dir: PathBuf::from("/usr/lib"),
             inc_dir: PathBuf::from("/usr/include"),
         })
+    } else if PathBuf::from("/root/.wasmedge/include/wasmedge")
```

Comment:
Please use the env variable of `HOME` or set a new env variable `WASMEDGE_DIR` to access the path.
The `/root` is a very specific and non-general value, so we should avoid this.

Reply:
Accepted. Thanks a lot!

Reply:
The issue has been fixed. Thanks!